LayoutTests/platform/gtk/TestExpectations

```diff
@@ -98,6 +98,9 @@ webkit.org/b/254733 inspector/canvas/shaderProgram-add-remove-webgl.html [ Failu
 # Crash on EWS bot.
 webrtc/vp8-then-h264-gpu-process-crash.html [ Skip ]
 
+# WebXR
+webkit.org/b/301551 imported/w3c/web-platform-tests/webxr/xrWebGLLayer_opaque_framebuffer.https.html [ Failure ]
+
```

Comment:
This is surprising. According to the bug it's failing with an incomplete attachment error. But the code is the same as the one used in vision which behaves as expected...

Reply:
It uses a subclass of the ANGLE GC.